src/main/java/mate/academy/util/HibernateUtil.java

```diff
@@ -0,0 +1,19 @@
+package mate.academy.util;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {
+    private static final SessionFactory instance = initSessionFactory();
+
+    public HibernateUtil() {
```

Comment:
```suggestion
    private HibernateUtil() {
```

Thus, no one will be able to create a new instance of the HibernateUtil class + we additionally show other developers that instances of HibernateUtil do not need to be created